src/convert.jl

```diff
@@ -71,6 +72,7 @@ convert_to_pm_type(::Type{<:Union{Base.Rational, Rational}}) = Rational
 convert_to_pm_type(::Type{<:Union{AbstractVector, Vector}}) = Vector
 convert_to_pm_type(::Type{<:Union{AbstractMatrix, Matrix}}) = Matrix
 convert_to_pm_type(::Type{<:Union{AbstractSparseMatrix, SparseMatrix}}) = SparseMatrix
+convert_to_pm_type(::Type{<:Union{AbstractSparseVector, SparseVector}}) = SparseVector
```

Comment:
we found a bug today with `IncidenceMatrix` which is converted to `SparseMatrix` could you please look at this?